app/views/NotificationPreferencesView/index.js

```diff
@@ -252,7 +252,7 @@ class NotificationPreferencesView extends React.Component {
 						theme={theme}
 					/>
 					<Separator theme={theme} />
-					<Info info={I18n.t('Receive_notifications_from', { name: room.name })} theme={theme} />
+					<Info info={I18n.t('Receive_notifications_from', { name: room.prid ? room.fname : room.name })} theme={theme} />
```

Comment:
Instead it, use `RocketChat.getRoomTitle`.

Reply:
@djorkaeffalexandre done

Reply:
@djorkaeffalexandre Please review